typeSummary.txt

```diff
@@ -69479,9 +69493,9 @@
      param httpMethod : com.microsoft.graph.http.HttpMethod
  class com.microsoft.graph.requests.extensions.GroupRequestBuilder : com.microsoft.graph.http.BaseRequestBuilder
    method acceptedSenders
-     return type com.microsoft.graph.requests.extensions.IDirectoryObjectCollectionRequestBuilder
+     return type com.microsoft.graph.requests.extensions.IDirectoryObjectCollectionWithReferencesRequestBuilder

```

Comment:
@zengin This could be considered a breaking change but I guess it wasn't working before this change and that's why you added a fix for metadata? https://github.com/microsoftgraph/msgraph-metadata/pull/58

Reply:
discussed on the dotnet equivalent PR, this is not breaking as it was broken to begin with (nobody could've taken a dependency on it)